.github/release.yml

```diff
@@ -0,0 +1,26 @@
+changelog:
```

Comment:
This only works if all the PRs have labels. Looks like there's not a workflow that automatically applies labels and most of the PRs that have been merged don't have labels.

Reply:
You're right.  We haven't been adding labels so far but I think the plan is to follow the same procedure used by kic and to start adding labels to the MRs so we can use the draft releases as a starting point for the release notes assuming some editing will be necessary.  @pleshakov Is that right?

Reply:
(discussed in a meeting)

yep, we want to labels.  this is a good label list standardized across the integration project 
https://github.com/nginxinc/nginx-plus-go-client/labels   ( we don't  need python label )

Reply:
Okay.  Thanks!  I copied this list from kubernetes-ingress into nginx-kubernetes-gateway/.github/release.yml which will be used by the code to determine which category to use in the draft release notes based on the label.  Do you want me to add the rest from https://github.com/nginxinc/nginx-plus-go-client/labels?   I would just need to know which category to put them in.

changelog:
  exclude:
    labels:
      - skip-changelog
  categories:
    - title: 🚀 Features
      labels:
        - enhancement
    - title: 💣 Breaking Changes
      labels:
        - change
    - title: 🐛 Bug Fixes
      labels:
        - bug
    - title: 📝 Documentation
      labels:
        - documentation
    - title: 🧪 Tests
      labels:
        - tests
    - title: 🔨 Maintenance
      labels:
        - chore
    - title: ⬆️ Dependencies
      labels:
        - dependencies
~

Reply:
@pammecrandall 
> Do you want me to add the rest from https://github.com/nginxinc/nginx-plus-go-client/labels? I would just need to know which category to put them in.

While we want to have those labels available, PRs with those labels are not needed in the changelog. Most of those remaining labels are used on issues, not PRs anyway.

Reply:
Okay.  I won't add them to the categories in release.yml but I just added them here.  https://github.com/nginxinc/nginx-kubernetes-gateway/issues/labels?sort=name-asc. Thanks!